.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+jobs:
+  build:
+     docker:
+      - image: python:2.7
+     working_directory: ~/repo
+     steps:
+       - checkout
+       - run:
+           name: Install dependencies
+           command: apt-get update && apt-get install libtool m4 automake npm -y
```

Comment:
we used to handle this sort of thing in `travis/deps.sh`, but when they switched to their new container setup, we had to move the package installation info to `.travis.yml`.   It would be good to change the `command:` here to something like:

```
   command: apt-get update && ./travis/deps.sh
```

and in `travis/deps.sh`:

```
case `uname` in
   Linux)
      apt-get install <INSERT LIST OF PACKAGES FROM TRAVIS.YML HERE>
```

That would allow folks to run `travis/deps.sh` when they want to make sure the pre-requisites are all installed, even when they are not on travis.